src/storage/config.rs

```diff
@@ -105,6 +105,14 @@ pub struct Config {
     #[online_config(skip)]
     pub txn_status_cache_capacity: usize,
     pub memory_quota: ReadableSize,
+    /// Maximum max_ts deviation allowed from PD timestamp (in seconds)
+    #[online_config(skip)]
```

Comment:
This configurations need to be changed online.

Reply:
Making the sync interval dynamically adjustable would add unnecessary complexity, while I've already made the other two parameters configurable at runtime.